src/promptflow/promptflow/_sdk/entities/_run.py

```diff
@@ -383,7 +384,7 @@ def _generate_run_name(self) -> str:
             flow_name = self._get_flow_dir().name
             variant = self.variant
             timestamp = datetime.datetime.now().strftime("%Y%m%d_%H%M%S_%f")
-            variant = parse_variant(variant)[1] if variant else "default"
+            variant = parse_variant(variant)[1] if variant else DEFAULT_VARIANT
```

Comment:
why is it "variant_0" even if there is no variant at all? Is this to align with portal name pattern?

Reply:
Yes